### DISCUSSION TITLE
Question  on " Plan did not converge to a solution."

### DISCUSSION THREAD

**Discussion by zhangsanfeng86** [NVlabs/curobo#15] [Software Q&A] [answered]
Hi,
@balakumar-s 
When run the demos, got this warning?
What's the meaning?
![image](https://github.com/NVlabs/curobo/assets/73547638/17c5c629-c917-4df7-a3b9-8388416a651a)

**Comment by balakumar-s** [comment, marked as answer]
That means a motion plan could not be computed for moving the robot from the current state to the goal pose. This can happen due to setting a unreachable goal pose or a goal pose in collision.

**Reply by zhangsanfeng86** [reply]
Thank you for your reply!

And I get confuse:  if set the number 2 to 20(for example),   it's more possible to get the warning!
https://github.com/NVlabs/curobo/blob/f2eb5f937afef065214daab7e09a56e8f7880176/examples/isaac_sim/simple_stacking.py#L484C30-L484C30
